analyses/maven/matcher.go

```diff
@@ -0,0 +1,67 @@
+package maven
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	_ driver.StackAnalysisInterface = (*Matcher)(nil)
+)
+
+// Matcher is State Object for Maven
+type Matcher struct {
+	FilePath string
```

Comment:
I might be wrong, but couldn't find the usage of `FilePath`

Reply:
Ack.